grakn-factory/src/main/java/ai/grakn/kb/internal/GraknTxJanus.java

```diff
@@ -74,7 +75,11 @@ public int numOpenTx() {
 
     @Override
     public void clearGraph() {
-        JanusGraphCleanup.clear(getTinkerPopGraph());
+        try {
+            JanusGraphCleanup.clear(getTinkerPopGraph());
+        } catch (BackendException e) {
+            e.printStackTrace();
```

Comment:
Is it perhaps better to rethrow: `throw new RuntimeException(e)` ?

Reply:
yes it’s a wip - thats why nobody is still assogned. also there is a PR waiting for you